src/pe_reports/pages.py

```diff
@@ -0,0 +1,150 @@
+"""Collect and distribute graphical data to readable charts in the presentation."""
+
+# Third-Party Libraries
+import pandas as pd
+from pptx.chart.data import CategoryChartData
+from pptx.enum.chart import XL_CHART_TYPE
+from pptx.util import Inches
+from stylesheet import Graph, Paragraph
```

Comment:
This needs to be a relative import.

```suggestion
from .stylesheet import Graph, Paragraph
```

Reply:
Resolved in 953dc55